3573/3/controllers/places_controller.rb

```diff
@@ -0,0 +1,13 @@
+class PlacesController < ApplicationController
+  get '/' do
+    @places = Place.all
+    erb :'/index'
+  end
+
+  get '/place/:id' do
+    @place = Place.find(params['id'])
+    @comments = @place.comments
+    @users = @comments.map(&:user)
```

Comment:
this will lead to n+1, you need to include users in the query for comments.

Reply:
If I understand correctly, I will solve this problem in show.erb :@comments.zip(@users)

Reply:
show log output to proof it.